backends/ebpf/CMakeLists.txt

```diff
@@ -222,10 +222,10 @@ else()
 endif()
 
 # List of executable dependencies
-set(TEST_DEPENDENCY_PROGRAMS tcpdump)
+set(TEST_DEPENDENCY_PROGRAMS gcc-multilib)
```

Comment:
Have you checked whether this actually works? I think you need to do a different check to make sure this is installed.